frame/elections-phragmen/src/lib.rs

```diff
@@ -115,6 +115,47 @@ type BalanceOf<T> =
 type NegativeImbalanceOf<T> =
 	<<T as Trait>::Currency as Currency<<T as frame_system::Trait>::AccountId>>::NegativeImbalance;
 
+
+/// Helper functions for migrations of this module.
+pub mod migrations {
+	use super::*;
+	use frame_support::migration::StorageKeyIterator;
+	use frame_support::Twox64Concat;
+
+	/// Migrate from the old legacy voting bond (fixed) to the new one (per-vote dynamic).
+	///
+	/// Will only be triggered if storage version is V1.
+	pub fn migrate_to_recorded_deposit<T: Trait>(old_deposit: BalanceOf<T>) -> Weight {
+		if <Module<T>>::pallet_storage_version() == StorageVersion::V1 {
```

Comment:
We could try out the pallet storage version (see https://github.com/paritytech/substrate/pull/7208) here.
Note, though, that the version would be `None` for now.

Reply:
mmh, just tried this and it seems complicated. Probably to better to wait with using `PalletVersion` until it is set in storage.

Reply:
> Probably to better to wait with using PalletVersion until it is set in storage?

this PR won't be part of v26 anymore thus we can make use of them now.